lib/rubocop/cop/rspec/factory_bot/dynamic_attribute_defined_statically.rb

```diff
@@ -62,10 +67,20 @@ def method_uses_parens?(location)
             location.begin.source == '(' && location.end.source == ')'
           end
 
-          def autocorrect_replacing_parens(node)
+          def value_is_inferred_hash?(value)
+            value.type == :hash && value.location.begin.nil?
```

Comment:
`value.hash_type? && value.location.begin.nil?` but what is an inferred hash anyway?
maybe you want to check for `value.hash_type? && value.braces?`

Reply:
Great, thanks! Still learning the node helpers 👍 Although I'm thinking you mean `value.hash_type? && !value.braces?` ?

By inferred hash I was attempting to describe a hash defined without braces (ie as a function parameter).

`foo(bar: 'baz')` vs `foo({ bar: 'baz' })`

The issue being, if the hash is defined without braces, we need to give it some when correcting the result and putting it into a block

Reply:
Changing the method name to `value_hash_without_braces?` to avoid confusion